sdk/textanalytics/azure-ai-textanalytics/swagger/authoring/README.md

```diff
@@ -0,0 +1,274 @@
+        "schema": {
+          "$ref": "#/definitions/TextAnalysisAuthoringImportProjectJobState"
+        }
+      };
+  - where-operation: TextAnalysisAuthoring_Train
+    transform: >
+      $["responses"]["200"] = {
+        "description": "dummy schema to get poller response when calling .result()",
+        "schema": {
+          "$ref": "#/definitions/TextAnalysisAuthoringTrainingJobState"
+        }
+      };
+  - where-operation: TextAnalysisAuthoring_DeployProject
+    transform: >
+      $["responses"]["200"] = {
+        "description": "dummy schema to get poller response when calling .result()",
+        "schema": {
+          "$ref": "#/definitions/TextAnalysisAuthoringDeploymentJobState"
+        }
+      };
+  - where-operation: TextAnalysisAuthoring_SwapDeployments
+    transform: >
+      $["responses"]["200"] = {
+        "description": "dummy schema to get poller response when calling .result()",
+        "schema": {
+          "$ref": "#/definitions/TextAnalysisAuthoringDeploymentJobState"
+        }
+      };
+  - where-operation: TextAnalysisAuthoring_DeleteDeployment
+    transform: >
+      $["responses"]["200"] = {
+        "description": "dummy schema to get poller response when calling .result()",
+        "schema": {
+          "$ref": "#/definitions/TextAnalysisAuthoringDeploymentJobState"
+        }
+      };
+  - where-operation: TextAnalysisAuthoring_CancelTrainingJob
+    transform: >
+      $["responses"]["200"] = {
+        "description": "dummy schema to get poller response when calling .result()",
+        "schema": {
+          "$ref": "#/definitions/TextAnalysisAuthoringTrainingJobState"
+        }
+      };
+```
+
+## Rename `body` param for operations
+
+```yaml $(tag) == 'release_authoring_2022_05_01'
+directive:
+  - where-operation: TextAnalysisAuthoring_CreateProject
+    transform: >
+        $.parameters[1]["x-ms-client-name"] = "project_parameters";
+  - where-operation: TextAnalysisAuthoring_Import
+    transform: >
+        $.parameters[1]["x-ms-client-name"] = "project_data";
+  - where-operation: TextAnalysisAuthoring_Train
+    transform: >
+        $.parameters[1]["x-ms-client-name"] = "training_parameters";
+  - where-operation: TextAnalysisAuthoring_SwapDeployments
+    transform: >
+        $.parameters[1]["x-ms-client-name"] = "deployment_names";
+  - where-operation: TextAnalysisAuthoring_DeployProject
+    transform: >
+        $.parameters[2]["x-ms-client-name"] = "deployment_label";
+```
+
+## Remove status operations
+
+```yaml $(tag) == 'release_authoring_2022_05_01'
+directive:
+  - remove-operation: TextAnalysisAuthoring_GetDeploymentStatus
+  - remove-operation: TextAnalysisAuthoring_GetSwapDeploymentsStatus
+  - remove-operation: TextAnalysisAuthoring_GetExportStatus
+  - remove-operation: TextAnalysisAuthoring_GetImportStatus
+  - remove-operation: TextAnalysisAuthoring_GetTrainingStatus
+  - remove-operation: TextAnalysisAuthoring_GetProjectDeletionStatus
+```
```

Comment:
Recent architect feedback for Language was to keep these status operations exposed in the client library.